rascaline-c-api/CMakeLists.txt

```diff
@@ -175,46 +155,88 @@ else()
     set(STRIP_ARGS "")
 endif()
 
+if (RASCALINE_BUILD_FOR_PYTHON)
+    # strip dynamic library for smaller wheels to download/install
+    set(CARGO_RUSTC_ARGS "${CARGO_RUSTC_ARGS};-Cstrip=symbols")
+endif()
+
 add_custom_target(cargo-build-rascaline ALL
     COMMAND ${CMAKE_COMMAND} -E env
         MACOSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}
-        RUSTFLAGS="${RUST_FLAGS}"
+        RUSTFLAGS=${RUST_FLAGS}
         cargo rustc ${CARGO_BUILD_ARG} -- ${SONAME_ARGS} ${STRIP_ARGS}
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
     DEPENDS ${ALL_RUST_SOURCES}
     COMMENT "Building ${RASCALINE_LIB_NAME} with cargo"
+    VERBATIM
+)
+
+add_custom_command(TARGET cargo-build-rascaline
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${RASCALINE_CAPI_LOCATION} ${RASCALINE_LOCATION}
+    DEPENDS cargo-build-rascaline
 )
 
 add_dependencies(rascaline cargo-build-rascaline)
 set(RASCALINE_HEADERS
-    "${PROJECT_SOURCE_DIR}/include/equistore.h"
     "${PROJECT_SOURCE_DIR}/include/rascaline.h"
-    "${PROJECT_SOURCE_DIR}/include/equistore.hpp"
     "${PROJECT_SOURCE_DIR}/include/rascaline.hpp"
 )
 set(RASCALINE_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include/)
 
+set(EQUISTORE_GIT_VERSION "6c5a833")
+if (RASCALINE_FETCH_EQUISTORE)
+    message(STATUS "Fetching equistore @ ${EQUISTORE_GIT_VERSION} from github")
+
+    if (CMAKE_VERSION VERSION_LESS 3.11)
```

Comment:
maybe declare as 3.10 directly here? Since 3.10 is the latest supported version.

Reply:
You mean `CMAKE_VERSION VERSION_EQUAL 3.10`? I feel like this is a bit clearer: we are adding a 3.11 feature to cmake <3.11